spec/parsers/maven_spec.rb

```diff
@@ -272,30 +272,10 @@
     expect(guavas[0][:requirement]).to eq '25.1-jre'
   end
 
-  it 'has the correct sections and dependencies from gradle-dependencies-q.txt' do
```

Comment:
I'd think this test still matters

Reply:
I thought since we're ignoring the section when deduping I'd just drop it but I'll modify it to just look for those section headers rather than looking for a specific dependency.